frontend/src/components/MemberProfile.tsx

```diff
@@ -0,0 +1,196 @@
+import { useState, useEffect } from 'react'
+import {
+	Button,
+	Flex,
+	Group,
+	Modal,
+	MultiSelect,
+	Select,
+	Stack,
+	Textarea,
+	TextInput
+} from '@mantine/core'
+import { Member } from 'types'
+
+interface MemberProfileProps {
+	opened: boolean
+	onClose: () => void
+	member: Member
+}
+
+const types = ['Undergraduate', 'Graduate', 'Alumni', 'Other']
+
+const faculties = [
+	'Arts',
+	'Engineering',
+	'Environment',
+	'Health',
+	'Math',
+	'Science'
+]
+
+const instruments = [
```

Comment:
Can we change these either to instruments or to be empty for now? And add a TODO for [CL-42](https://uwcbc.atlassian.net/browse/CL-42)